packages/browser-wallet/src/popup/shared/utils/identity-helpers.ts

```diff
@@ -8,3 +9,71 @@ export function useGetAttributeName() {
         return attributeKey in sharedTranslations.idAttributes ? t(attributeKey) : attributeKey;
     };
 }
+
+enum Sex {
+    NotKnown,
+    Male,
+    Female,
+    NA = 9,
+}
+
+function formatGender(sex: Sex, t: TFunction<'shared', 'idAttributeValues'>) {
+    switch (sex) {
+        case Sex.NotKnown:
+            return t('sex.NotKnown');
+        case Sex.Male:
+            return t('sex.Male');
+        case Sex.Female:
+            return t('sex.Female');
+        case Sex.NA:
+            return t('sex.NA');
+        default:
+            return t('sex.Unavailable');
+    }
+}
+
+enum DocumentType {
+    NA,
+    Passport,
+    NationalIdCard,
+    DriversLicense,
+    ImmigrationCard,
+}
+
+function formatDocumentType(value: string, t: TFunction<'shared', 'idAttributeValues'>) {
+    const type = parseInt(value, 10);
+
+    switch (type) {
+        case DocumentType.NA:
+            return t('documentType.NA');
+        case DocumentType.NationalIdCard:
+            return t('documentType.NationalIdCard');
+        case DocumentType.Passport:
+            return t('documentType.Passport');
+        case DocumentType.DriversLicense:
+            return t('documentType.DriversLicense');
+        case DocumentType.ImmigrationCard:
+            return t('documentType.ImmigrationCard');
+        default:
+            return value;
+    }
+}
+
+export function useDisplayAttributeValue() {
+    const { t, i18n } = useTranslation('shared', { keyPrefix: 'idAttributeValues' });
```

Comment:
You don't need to make this a hook, you can access the text like this :thinking:
```
import i18n from '@popup/shell/i18n';
...
    return i18n.t('idAttributeValues.documentType.NA');
```

Reply:
If we do that, then we have to check whether translations are actually loaded. See https://github.com/i18next/react-i18next/issues/909. So I believe the hook way is probably simpler.